src/ec/packages/ec-component-menu-legacy/docs/menu-legacy.md

```diff
@@ -0,0 +1,46 @@
+# ECL Twig - EC Navidation Menu
+
+npm package: `@ecl-twig/ec-component-navigation-menu`
```

Comment:
component name should be updated here (and next line too)

Reply:
ok. I also write the right example...